cpp/include/algorithms.hpp

```diff
@@ -587,6 +595,8 @@ void sssp(GraphCSRView<VT, ET, WT> const &graph,
  *
  * @param[in] directed               Treat the input graph as directed
  *
+ * @param[in] mg_batch               If set to true use SG BFS path when comms are initialized.
+ *
  * @throws     cugraph::logic_error when an error occurs.
```

Comment:
This repeats 

https://github.com/rapidsai/cugraph/pull/940/files#diff-5fe5e7f6e43a9f3405cdb72bb11fa4caR572

Reply:
Updated in feb6ff4